### ISSUE TITLE
Various

### ISSUE THREAD

**Issue by danbaz101** [DerJantob/TSW2_Controller#53]
Hi Jannik,

Hope you are well.
Thanks for the further development you have done on TSW2 Controller!
Currently I am back to Train simming and have purchased TSW3 and trying to get it setup with your program!
Doing so I am running into a few issues.
Are you still providing support for this program?

1. Convert old to new config file instructions. When I installed the new version I saw the instructions, did not have time to do it right away and then could not locate the instructions any more. Where should I look for these?

2. In controls section the Zuruck button still in German language = minor detail, but wanted to make you aware.

3. In Edit train section, the buttons that you click on in the video (-100, 0 and 100), do nothing when I press them. I can add some info in the edit window.

4. In the video when you move a slider you see in the edit window the result, how it is read above the moving indicator (the green band). This does not work at all on my end.

5. I am trying to get the brake to work on the ICE1, basically the same lever as in the 146 that you demo'ed, but I cannot get it to work correctly.
     5.1 From -100 to 0 nothing happens
     5.2 From 0-10 or so, then one nothch of brakes is selected, moving further nothing happens
     5.3 When going back to 0 - -100 section stil nothing happens
     5.4. Moving to 0 - 10 it moved a second notch.
This my setup for this Brake lever:
BR401,,Brake,1,RotX,-100|-100|100|100,,,,,Stufen,9,[14=1][Running=0][Release=0][1A=1][1B=2][2=3][3=4][4=5][5=6][6=7][7=8][Full Service=9],125,

I will leave it with this for now.

Hope to hear from you.

Kind regards,

Daniel

**Comment by DerJantob** [comment]
Hey,
Although I'm no longer actively working on this program, if someone contacts me, I'll certainly do my best to help.
I'm not entirely sure at the moment, but I think the guide for converting the configuration should be on your desktop in the "Update_Info" folder. However, I'll resend the instructions shortly. 
I'm a bit busy in the next few days, but if I find the time, I'll definitely take a look at resolving the issues you mentioned. 
If you notice any further issues in that time, please feel free to reach out again!


Kind regards
Jannik

_
_
_

Here are the instructions:
```
A lot has changed in version 2.0.0. Among other things, the way controllers work.
Since I decided to treat Throttle and Master Controller separately, I can't just copy the old text indicators.

So what do I need to do to transfer my old settings to the new version?

1. 
Go to \"Settings\"->\"Controls\"->\"Global keybinds\".
There you should now find the controllers Throttle, Brake and Master Controller.

2. 
Select Throttle and check the keybinds first.
Next check the main indicators.
But you should no longer enter the text indicators for the Master Controller here!
The text indicators you had before for the trottle and Master Controller are the following:


3.
Do the same for the brake.
The text indicators you had before for the brake are the following:


4.
Now select \"Master Controller\". As you can see, the checkbox \"Master Controller\" is checked.
This allows the controller to also go into the braking area (negative numbers).
Here you now enter the indicators for the Master Controller and those for the throttle/ and brake area.
You previously had the following indicators:
Throttle area:

Brake area:


Done!
```

**Comment by danbaz101** [comment]
Hi Jannik,

I appreciate the help you are giving and will do all I can help you in that too.
The folder that you mentioned is not on my system. I have followed the steps that you have provided above and I needed this to rename the oem3 to oem1 for brake decrease, so this fixed part of the problem. Thank you!

The problem for topic 5 changed like this:
5.1 From -100 to 0 nothing happens, so half the travel of the lever is not used.
5.2 From 0-100 tis is where all the 9 notches are applied both ways.

In this snip you see ->0 above the box. Could this have something to do with it? 
![image](https://github.com/DerJantob/TSW2_Controller/assets/83304085/5eb25b45-743b-4315-a0a9-ec35fed89674)

Kind regards,

Daniel

**Comment by danbaz101** [comment]
Hello Jannik,

Although I still am puzzled why for topic 5 only half the lever movement is used and would still like to know how I can adjust that myself for other situations, I have found a good interim solution for the BR401. I am using button as a lever to Release the brakes when <-50 which works. The brakes are set then between 0 and 100, so I have 0 - -50 as the zero position.

Kind regards,

Daniel

**Comment by danbaz101** [comment]
Topics from this thread have been solved through pre release version 2.0.5
